app/components/Text.tsx

```diff
@@ -83,21 +97,24 @@ const $sizeStyles = {
   xxs: { fontSize: 12, lineHeight: 13.2 } as TextStyle,
 }
 
-const $fontWeightStyles = Object.entries(typography.primary).reduce((acc, [weight, fontFamily]) => {
-  return { ...acc, [weight]: { fontFamily } }
-}, {}) as Record<Weights, TextStyle>
+const $fontWeightStyles = Object.entries(typography.primary).reduce(
+  (acc, [weight, fontFamily]) => {
+    return { ...acc, [weight]: { fontFamily } }
+  },
+  {}
+) as Record<Weights, TextStyle>
 
 const $baseStyle: StyleProp<TextStyle> = [
   $sizeStyles.sm,
   $fontWeightStyles.book,
-  { color: colors.text },
+  { color: colors.text, lineHeight: 22.4 },
```

Comment:
[TOL] `22.4` seems specific? is there are a value we can use from spacing or fonts?

Reply:
[TOL] Was this meant to be applied to the base style? The size/line height combos were taken from here, but there could be discrepancies throughout the designs too

https://www.figma.com/file/zKpQXwCdVWQc4yDWDY83Vw/cr-design-library?node-id=39%3A7&t=XS9BEQ2g8KA53R2l-0

Reply:
Ah yeah, good call out! I thought so too... however, adding it does make it if not identical to the designs 🤔 

Looking at the designs again and shuffling through each page and clicking on textual elements, it seems like default text is either `22.4` or `22` -- let me ask Justin what we should do here.

_honestly, I think working with only default line heights and baking everything into height is perhaps better. Because design applications generated css these days, it seems like this fluctuation of something like `line-height` could be easily overlooked_

Reply:
>~honestly, I think working with only default line heights and baking everything into height is perhaps better.~ Because design applications generated css these days, it seems like this fluctuation of something like line-height could be easily overlooked

Take that back 😆  line-heights are still needed for paragraph text especially. If I remember correctly, I think we could create in Figma preset design tokens for all text elements